app/src/main/java/com/example/tung/demoretrofit/sceen/MainActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.example.tung.demoretrofit.sceen;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import com.example.tung.demoretrofit.R;
+import com.example.tung.demoretrofit.data.model.SubMovie;
+import com.example.tung.demoretrofit.data.repository.MoveRepository;
+import com.example.tung.demoretrofit.data.source.MovieRemoteDataSource;
+import com.example.tung.demoretrofit.databinding.ActivityMainBinding;
+import com.example.tung.demoretrofit.sceen.ImageMovie.ImageMovieActivity;
+import com.example.tung.demoretrofit.sceen.ImageMovie.MovieNavigator;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity implements MovieNavigator {
+    private ActivityMainBinding mBinding;
+    private HomeViewModel mViewModel;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        initBinding();
+        initRecycler();
+    }
+
+    private void initBinding() {
+        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_main);
+        mViewModel = new HomeViewModel(MoveRepository.getInstance(MovieRemoteDataSource.getInstance()), this);
+        mBinding.setHomeViewModel(mViewModel);
+    }
+
+    private void initRecycler() {
+        HomeAdapter adapter = new HomeAdapter(new ArrayList<SubMovie>(), mViewModel);
+        RecyclerView recyclerView = mBinding.recyclerMovie;
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
```

Comment:
ngoài ra e cũng có thể set LayoutManager trực tiếp trong xml luôn cũng được nhé
```suggestion
        <android.support.v7.widget.RecyclerView
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            app:layoutManager="GridLayoutManager"
            app:spanCount="2"
            />
            
            hoặc
  
          <android.support.v7.widget.RecyclerView
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            app:layoutManager="LinearLayoutManager"
            />
```